react-task/src/App.js

```diff
@@ -1,25 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import Register from "./components/Register";
 
 function App() {
```

Comment:
Try follow the new standard for defining functional components going forward which utilises arrow functions

```suggestion
const App = () => {
```